tests/unit/test_steps.py

```diff
@@ -728,6 +728,74 @@ def test_change_to_unknown_resolution_with_resolution_map(
     ]
 
 
+def test_update_issue_priority(
```

Comment:
I'd like to see some tests an invalid priority and an invalid severity being provided. Just so we don't accidentally make a change in the future.